app/assets/javascripts/discourse/app/initializers/inject-discourse-objects.js

```diff
@@ -9,11 +9,10 @@ import User from "discourse/models/user";
 import { registerDiscourseImplicitInjections } from "discourse/lib/implicit-injections";
 
 export default {
-  name: "inject-discourse-objects",
   after: "discourse-bootstrap",
 
-  initialize(container, app) {
-    const siteSettings = container.lookup("service:site-settings");
+  initialize(app) {
+    const siteSettings = app.__container__.lookup("service:site-settings");
```

Comment:
This is again highly suspicious – the service will likely have an incorrect owner, and may get leaked/cached inappropriately beyond the application instance's lifetime.